src/gui/advancedsettings.h

```diff
@@ -34,6 +34,7 @@
 #include <QSpinBox>
 #include <QTableWidget>
 
+#include "gui/desktopintegration.h"
```

Comment:
Included for `QBT_USES_CUSTOMDBUSNOTIFICATIONS`. Also someone need to move that macro declaration from that file to cmake, qmake build scripts, it is ugly to include that header here.